internal/runtime/hcsv2/container.go

```diff
@@ -116,6 +118,16 @@ func (c *Container) Kill(ctx context.Context, signal syscall.Signal) error {
 	return nil
 }
 
+func (c *Container) DeleteContainerState(ctx context.Context) error {
```

Comment:
Lets just call this `Delete`

Reply:
Are you okay with leaving the bridge call as "deleteContainerState"?

Reply:
Lets call it `ComputeSystemDeleteV1`

Reply:
That matches the others better

Reply:
But really we dont need this func. We can do this automatically when the publish notification is done. Maybe we should just move to the ttrpc protocol for LCOW?